src/sentry/static/sentry/app/components/assigneeSelector.tsx

```diff
@@ -400,7 +400,7 @@ const AssigneeSelectorComponent = createReactClass<Props, State>({
               <DropdownButton {...getActorProps({})}>
                 {assignedTo ? (
                   <ActorAvatar actor={assignedTo} className="avatar" size={24} />
-                ) : suggestedActors ? (
+                ) : suggestedActors && suggestedActors.length ? (
```

Comment:
IMO I like to be explicit and use `length > 0`, but that's just me.

Reply:
i will change just for u evan